### ISSUE TITLE
basic navigation opens up expanded by default

### ISSUE THREAD

**Issue by ashishjoshi42** [siemens/ix#829]
### What happened?

The basic navigation opens up expanded by default. When you open a page with ix-basic-navigation, the page opens with the nav expanded which is not a good user experience.

### What type of frontend frameware are you seeing the problem on?

Angular

### Which version of iX do you use?

v2.0.1

### Code to produce this issue.

```shell
The issue can be seen in the basic navigation sample itself


Open this link in a new window

https://ix.siemens.io/webcomponent-examples/dist/preview-examples/basic-navigation.html?theme=theme-brand-dark

Expected: The navbar should not be expanded
Actual: The nav opens up expanded by default

Close the nav by clicking on the chevron. Now resize the window either by doing "restore" + "maximize" or even do a right click => Inspect.

Expected: the state of nav should not change.
Actual: The nav automatically gets expanded
```

**Comment by danielleroux** [comment]
Did you checkout the migration guide? https://ix.siemens.io/docs/migration/2_0_0/#multiple-screen-layouts-within-ix-basic-navigation

You can ajust this by defining the breakpoints you need.

**Comment by ashishjoshi42** [comment]
But why is the default that it opens expanded? Shouldn't it be closed by default for all screen sizes? 
Also is there a property by which you can override this default "expanded" mode?

**Comment by silviowolf** [comment]
Hi @ashishjoshi42, you are right, from an UX point of view the current behavior is unexpected and we will fix the default behavior to be collapsed. Nevertheless, we have also requirements where the default should be always expanded, whenever possibles (enough space). So we will make this behavior configurable.

**Comment by nuke-ellington** [comment]
Should be fixed in #1156
